lib/bsdlib/bsd_os.c

```diff
@@ -249,6 +414,11 @@ void trace_uart_init(void)
 /* This function is called by bsd_init and must not be called explicitly. */
 void bsd_os_init(void)
 {
+	sys_slist_init(&sleeping_threads);
+	atomic_clear(&rpc_event_cnt);
+
+	memset(thread_event_monitor, 0, sizeof(thread_event_monitor));
```

Comment:
This could be removed now that the array is static

Reply:
Done